acceptance/tests/reports/report_storage.rb

```diff
@@ -2,14 +2,14 @@
 
 test_name "basic validation of puppet report submission" do
 
-  step "setup a test manifest for the master and perform agent runs" do
-    manifest = <<-MANIFEST
+  manifest = <<-MANIFEST
```

Comment:
Although I can see what you've done at the end of this file. Somehow I think this should be inverted though, we've gone and modified an existing test with noop stuff, and then added back a non-noop test. I'd at least invert the way we're doing this here, the noop test should be the bit we've added at the end, not the other way around.